app/helpers/announcements_helper.rb

```diff
@@ -4,7 +4,8 @@ def announcement_hidden?(announcement)
   end
 
   def current_announcement
-    @current_announcement = Announcement.current
```

Comment:
I thought i suggested the short hand would be fine here. Do we need this more verbose style of memorization?